packages/toolpad-app/src/toolpad/Toolpad.tsx

```diff
@@ -55,18 +59,30 @@ export default function Toolpad({ basename, envVarNames }: ToolpadProps) {
   return (
     <NoSsr>
       <ToolpadContext.Provider value={toolpadContextValue}>
-        {/* Container that allows children to size to it with height: 100% */}
-        <Box sx={{ height: '1px', minHeight: '100vh' }}>
-          <ErrorBoundary fallbackRender={ErrorFallback}>
-            <React.Suspense fallback={<FullPageLoader />}>
-              <BrowserRouter basename={basename}>
-                <Routes>
-                  <Route path="/*" element={<AppEditor />} />
-                </Routes>
-              </BrowserRouter>
-            </React.Suspense>
-          </ErrorBoundary>
-        </Box>
+        <ThemeProvider>
+          <Head>
```

Comment:
I just removed this in https://github.com/mui/mui-toolpad/pull/1888
Is this a merge gone wrong?